spooq/transformer/null_cleaner.py

```diff
@@ -0,0 +1,168 @@
+import pyspark.sql.functions as F
+import pyspark.sql.types as sql_types
+from pyspark.sql.column import Column
+
+from .base_cleaner import BaseCleaner
+
+
+class NullCleaner(BaseCleaner):
+    """
```

Comment:
Docstring
Please also add the entry in the documentation and the imports. See ["Create your own transformer"](https://spooq.readthedocs.io/en/latest/base_classes/transformer.html#custom-transformer)

Reply:
Yep. Done.

Reply:
Could you please also update the hint in the docstring for the enum-cleaner?
"If you want to replace Null values you should use the method ~pyspark.sql.DataFrame.fillna from Spark." 
==> `` If you want to replace Null values you should use the transformer :py:class:`spooq.transformer.NullCleaner`. ``

Reply:
Done!